### PR TITLE
sql-statement: add ADMIN ddl statements

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
First batch of https://github.com/pingcap/docs/issues/2639

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3587 https://github.com/pingcap/docs/pull/4184
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

File: sql-statements/sql-statement-admin-cancel-ddl.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+>
+> + 如果希望取消的作业已经完成，取消操作将失败。
+
+## MySQL 兼容性
+
+`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。
+
+## 另请参阅
+
+* [`ADMIN SHOW DDL [JOBS,QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md)
```
Comment (djshow832) [inline sql-statements/sql-statement-admin-cancel-ddl.md:46]:
```suggestion
* [`ADMIN SHOW DDL [JOBS|QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md)
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+title: ADMIN SHOW DDL [JOBS|QUERIES]
+summary: TiDB 数据库中 ADMIN SHOW DDL [JOBS|QUERIES] 的使用概况。
+---
+
+# ADMIN SHOW DDL [JOBS,QUERIES]
+
+`ADMIN SHOW DDL [JOBS,QUERIES]` 语句显示了正在运行和最近完成的 DDL 作业的信息。
```
Comment (djshow832) [inline sql-statements/sql-statement-admin-show-ddl.md:8]:
```suggestion
# ADMIN SHOW DDL [JOBS|QUERIES]

`ADMIN SHOW DDL [JOBS|QUERIES]` 语句显示了正在运行和最近完成的 DDL 作业的信息。
```

**Review by djshow832 — COMMENTED** [review]
LGTM

File: sql-statements/sql-statement-admin-cancel-ddl.md (hunk 1 above)
Comment (lilin90) [reply sql-statements/sql-statement-admin-cancel-ddl.md]: @Joyinqin Please also confirm and update changes in these two comments by @djshow832 to the English version. Thanks!

**Comment by lilin90** [comment]
@Joyinqin Please take a look at my comment: https://github.com/pingcap/docs-cn/pull/4801#discussion_r518515659

**Comment by Joyinqin** [comment]
> @Joyinqin Please take a look at my comment: [#4801 (comment)](https://github.com/pingcap/docs-cn/pull/4801#discussion_r518515659)

I updated it in https://github.com/pingcap/docs/pull/4184, thx for the comment!

File: sql-statements/sql-statement-admin-cancel-ddl.md (hunk 1 above)
Comment (Joyinqin) [reply sql-statements/sql-statement-admin-cancel-ddl.md]: I updated it in https://github.com/pingcap/docs/pull/4184, and thx for the comment!

File: sql-statements/sql-statement-admin-checksum-table.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+![AdminStmt](/media/sqlgram/AdminStmt.png)
+
+**TableNameList:**
+
+![TableNameList](/media/sqlgram/TableNameList.png)
+
+## 示例
+
+计算表 `t1` 的校验和:
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-checksum-table.md:22]:
```suggestion
计算表 `t1` 的校验和：
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+- 对于破坏性的更改（例如删除索引或删除表），当作业完成时，`SCHEMA_STATE` 将变为 `none`。对于附加更改，`SCHEMA_STATE` 将变为 `public`。
+
+若要限制表中显示的行数，可以指定 `NUM` 和 `WHERE` 条件：
+
+```sql
+ADMIN SHOW DDL JOBS [NUM] [WHERE where_condition];
+```
+
+* `NUM`：查看已经执行完成的 DDL 作业队列中最近 `NUM` 条结果，未指定时，默认值为 10。
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-show-ddl.md:91]:
```suggestion
* `NUM`：用于查看已经执行完成的 DDL 作业队列中最近 `NUM` 条结果；未指定时，默认值为 10。
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+* `WHERE`：`WHERE` 子句，可以添加过滤条件。
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-show-ddl.md:92]:
```suggestion
* `WHERE`：`WHERE` 子句，用于添加过滤条件。
```

File: sql-statements/sql-statement-admin.md [hunk 6]
```diff
@@ -9,92 +9,12 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-admin/','/docs-cn/dev/refer
...
-
-```sql
-ADMIN CHECKSUM TABLE tbl_name [, tbl_name] ...;
-```
-
-以上语句会获取 `tbl_name` 的 64 位的 checksum 值，该值可通过计算了表中所有的键值对（包括行数据和索引数据）的 CRC64 获得。
+| 语句                                                                                | 功能描述                 |
+|------------------------------------------------------------------------------------------|-----------------------------|
+| [`ADMIN CANCEL DDL JOBS`](/sql-statements/sql-statement-admin-cancel-ddl.md)             | 取消当前正在运行的 DDL 作业 |
+| [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md)          | 计算表中所有行和索引的 CRC64C 校验和 |
```
Comment (lilin90) [inline sql-statements/sql-statement-admin.md:15]:
```suggestion
| [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md)          | 计算表中所有行和索引的 CRC64 校验和 |
```

File: sql-statements/sql-statement-admin-checksum-table.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+title: ADMIN CHECKSUM TABLE
+summary: TiDB 数据库中 ADMIN CHECKSUM TABLE 的使用概况。
+---
+
+# ADMIN CHECKSUM TABLE
+
+`ADMIN CHECKSUM TABLE` 语句用于计算表中所有行和索引的 CRC64C 校验和。在 TiDB Lightning 等程序中，可通过此语句来确保导入操作成功。
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-checksum-table.md:8]: CRC64C -> CRC64. Please be careful.
```suggestion
`ADMIN CHECKSUM TABLE` 语句用于计算表中所有行和索引的 CRC64 校验和。在 TiDB Lightning 等程序中，可通过此语句来确保导入操作成功。
```

**Comment by Joyinqin** [comment]
@lilin90 PTAL again, thank you~

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**